pkg/vpcmodel/output.go

```diff
@@ -115,6 +115,30 @@ func NewOutputGenerator(c1, c2 MultipleVPCConfigs, grouping bool, uc OutputUseCa
 	return res, nil
 }
 
+// UnificationDebugPrint used by testing to test unification
+func (o *OutputGenerator) UnificationDebugPrint() string {
```

Comment:
move  this function to the file `groupingUnification_test.go`

Reply:
It was already there. Forget to push. pushed now

Reply:
no change here yet...

Reply:
sorry, I did not read your comment properly. Answered something else.
Anyways,  the function is in `vpcmodel`; the testing is in `ibmvpc` - since it executes files from there.
It is possible to move the function to  `ibmvpc`  but is it worth it? it will require globalization groupedEndpointsElems.
Plus, I don't see a problem in keeping it in `unifyGrouping` - its small and the function is all about debugging unify grouping.

Reply:
ok